src/leaflike/bookmarks.clj

```diff
@@ -30,26 +27,27 @@
   [{:keys [params] :as request}]
   (let [bookmark (-> (select-keys params [:title :url :tags])
                      format-tags)
-        user    (get-user request)]
+        user    (hutils/get-user request)
+        tags (:tags bookmark)]
```

Comment:
@samrat Isn't `format-tag` applicable here instead of entire `bookmark`?

Reply:
I am using the formatted tags here. The `bookmark` let binding formats it.

Reply:
Fixed this. Since `tags` is now a separate entity and no longer stored in the bookmarks table, I've changed and renamed the fn and how it's used here.